### PR TITLE
[mingw-w64] Fix bad URL

### PR DESCRIPTION
**Pull request by EstebanDugueperoux2**
Specify library name and version:  **lib/1.0**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [x ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by ericLemanissier — DISMISSED** [review]

**Comment by uilianries** [comment]
@EstebanDugueperoux2 please, consider https://github.com/EstebanDugueperoux2/conan-center-index/pull/1

**Review by prince-chrismc — DISMISSED** [review]

**Comment by EstebanDugueperoux2** [comment]
> @EstebanDugueperoux2 please, consider [EstebanDugueperoux2#1](https://github.com/EstebanDugueperoux2/conan-center-index/pull/1)

Hi @uilianries I don't understand what you want I do.

**Comment by uilianries** [comment]
> > @EstebanDugueperoux2 please, consider [EstebanDugueperoux2#1](https://github.com/EstebanDugueperoux2/conan-center-index/pull/1)
> 
> Hi @uilianries I don't understand what you want I do.

@EstebanDugueperoux2 Fix linter errors

File: recipes/mingw-w64/linux/conanfile.py [hunk 1]
```diff
@@ -215,18 +224,18 @@ def build(self):
...
 
         self.output.info("Building done!")
 
     def package(self):
         self.copy("COPYING", src=os.path.join(self.build_folder, "sources", "mingw-w64"), dst="licenses")
-        tools.remove_files_by_mask(self.package_folder, "*.la")
-        tools.rmdir(os.path.join(self.package_folder, "share", "man"))
-        tools.rmdir(os.path.join(self.package_folder, "share", "doc"))
-        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        rm(self, "*.la", self.package_folder)
```
Comment (uilianries) [inline recipes/mingw-w64/linux/conanfile.py:235]:
```suggestion
        rm(self, "*.la", self.package_folder, recursive=True)
```
It should remove those files listed by the hook.

**Comment by EstebanDugueperoux2** [comment]
Hi @uilianries, 

Is it ok now?

**Review by uilianries — APPROVED** [review]
LGTM

**Comment by uilianries** [comment]
@EstebanDugueperoux2 It's good enough! Thank for your contribution!

**Review by toge — APPROVED** [review]

**Pull request merged**